File: C++/747. Largest Number At Least Twice of Others.cpp
class Solution {
public:
    int dominantIndex(vector<int>& n) {
        int i,m=-1,k;
        for(i=0;i<n.size();i++){
            if(m<n[i]){
                m=n[i];
                k=i;
            }
        }
        sort(n.rbegin(),n.rend());
        if(n[0]<2*n[1])
        return -1;
        return k;
    }
};

File: JAVA/747. Largest Number At Least Twice of Others.java
class Solution {
    public int dominantIndex(int[] n) {
        int i,k=-1,m=-1;
        for(i=0;i<n.length;i++){
            if(m<n[i]){
                m=n[i];
                k=i;
            }
        }
        Arrays.sort(n);
        if(n[n.length-1]>=2*n[n.length-2])
        return k;
        return -1;
    }
}